Gemfile.lock

```diff
@@ -72,9 +72,9 @@ GEM
     coderay (1.1.3)
     concurrent-ruby (1.1.8)
     crass (1.0.6)
-    dedup (0.1.2)
+    dedup (0.1.3)
```

Comment:
My `bundle install` modified `dedup` and `ffi`, not sure if we should keep or remove.

Reply:
Let's remove both random gem changes.